tests/test_helpers.py

```diff
@@ -473,62 +473,35 @@
       "--group-members", "ascopatz", "--grants", "SymPy-1.1", "--due-date", "2021-01-01", '--notes', 'new note'],
-     "ly_newprojectum has been added in projecta\n"),
-    (["helper", "u_milestone", "sb_firstprojectum", "--index", "4,5",
-      "--status", "converged", "--due-date", "2020-06-01", "--notes", "do this",
-      "do that", "--type", "meeting"],
-     "sb_firstprojectum has been updated in projecta\n"
-     ),
-    (["helper", "u_milestone", "sb"],
-     "Projecta not found. Projecta with similar names: \n"
-     "sb_firstprojectum\n"
-     "Please rerun the helper specifying the complete ID.\n"
+     "ly_newprojectum has been added in projecta\n"
      ),
-    (["helper", "u_milestone", "sb_firstprojectum"],
-     "Please choose from one of the following to update/add:\n"
-     "1. new milestone\n"
-     "2. Kick off meeting    due date: 2020-05-06    status: finished\n"
-     "3. Project lead presentation    due date: 2020-05-20    status: proposed\n"
-     "4. deliverable    due date: 2020-06-01    status: converged\n"
-     "5. planning meeting    due date: 2020-06-01    status: converged\n"
-     ),
-    (["helper", "u_milestone", "sb_firstprojectum", "--verbose"],
-     "Please choose from one of the following to update/add:\n"
-     "1. new milestone\n"
-     "2. Kick off meeting    due date: 2020-05-06    status: finished\n"
-     "     audience: ['lead', 'pi', 'group_members']\n"
-     "     objective: introduce project to the lead\n"
-     "     type: meeting\n"
-     "     notes:\n"
-     "       - kickoff note\n"
-     "3. Project lead presentation    due date: 2020-05-20    status: proposed\n"
-     "     audience: ['lead', 'pi', 'group_members']\n"
-     "     objective: lead presents background reading and initial project plan\n"
-     "     type: meeting\n"
-     "     notes:\n"
-     "       - do background reading\n"
-     "       - understand math\n"
-     "4. deliverable    due date: 2020-06-01    status: converged\n"
-     "     audience: ['beginning grad in chemistry']\n"
-     "     type: meeting\n"
-     "     notes:\n"
-     "       - deliverable note\n"
-     "       - do this\n"
-     "       - do that\n"
-     "5. planning meeting    due date: 2020-06-01    status: converged\n"
-     "     audience: ['lead', 'pi', 'group_members']\n"
-     "     objective: develop a detailed plan with dates\n"
-     "     type: meeting\n"
-     "     notes:\n"
-     "       - do this\n"
-     "       - do that\n"
-     ),
-    (["helper", "u_milestone", "sb_firstprojectum", "--current"],
-     "Please choose from one of the following to update/add:\n"
-     "1. new milestone\n"
-     "3. Project lead presentation    due date: 2020-05-20    status: proposed\n"
-     "4. deliverable    due date: 2020-06-01    status: converged\n"
-     "5. planning meeting    due date: 2020-06-01    status: converged\n"
+    (["helper", "u_milestone", "--milestone_uuid", "milestone_uuid_sb1_", "sb_firstpro", "kosb_firstpro",
```

Comment:
Make sure your test does as much work as it can.  Is this updating everything that it can?  I see a bunch of updates not covered by tests above and I think they could be?  In principle, we can use `u_milestone` to update every item in a milestone, so the test can cover everything (but not creating a new milestone because of our clunky testing structure)